components/Table/Table.vue

```diff
@@ -26,13 +37,40 @@ export default {
     items: {
       type: Array,
       required: true
+    },
+    showExpand: Boolean
+  },
+  data() {
+    return {
+      toggle: [],
+      expanded: []
+    };
+  },
+  computed: {
+    columnSpan() {
+      return this.showExpand ? this.headers.length + 1 : this.headers.length;
     }
   },
   methods: {
     textAlign(align) {
       return {
         "text-align": align || "left"
       };
+    },
+    expandDetail(id) {      
```

Comment:
this function doesn't only expand but also collapse so should be better to call it `toggleXX`

Reply:
I changed from expandDetail to toggleItem.